bridge/client/styles.scss

```diff
@@ -426,7 +426,7 @@ pre.code {
   margin: 0 5px;
 }
 
-.dt-icon.event-icon {
+.dt-icon {
```

Comment:
Why did you remove `.event-icon` here? This will affect a lot of components. Are you sure this does not cause any incorrect styles?

Reply:
Then the `event-icon` class is useless and can be removed for all components

Reply:
That is true, I will remove it :)